src/Accounts/Account.java

```diff
@@ -0,0 +1,77 @@
+package Accounts;
```

Comment:
[Fix] numele de pachete in java sunt cu litera mica, doar numele de Clase incep cu litera mare. De asemenea, toate pachetele trebuie sa vina sub cel al oragnizatiei/companiei pt care este proiectul. In general se foloste com.example, sau com.company, cand nu ai o companie adevarata